charts/seed-bootstrap/charts/fluent-bit/values.yaml

```
@@ -33,3 +33,6 @@ lokiLabels:
     hostname: "host_name"
     unit: "systemd_component"
 exposedComponentsTagPrefix: "user-exposed"
+
+eventLogger:
+  enabled: false
```

Nit: you could leave 1 empty line at the end of the file.

```suggestion
  enabled: false

```